src/test/java/com/google/api/codegen/testdata/discoveries/php/php_translate.v2.json.baseline

```diff
@@ -15,18 +15,12 @@ require_once __DIR__ . '/vendor/autoload.php';
 
 $client = new Google_Client();
 $client->setApplicationName('Google-TranslateSample/0.1');
-
-// TODO: Change placeholder below to desired API key:
-$client->setDeveloperKey('{MY-API-KEY}');
+$client->setDeveloperKey(''); // TODO: Update placeholder with desired API key.
 
 $service = new Google_Service_Translate($client);
 
-
-// TODO: Change placeholder below to desired parameter value for the `listDetections` method:
-
 // The text to detect
-$q = [];
-
+$q = array();  // TODO: Update placeholder value.
```

Comment:
Hmm. I guess I had written `array()` for `optParams` since it was being used with just named indices, but now that you mention it, maybe it would be more idiomatic to use `[]` for both. Certainly for the placeholder; what do you think about `optParams`?